backend/src/main/resources/application.properties

```diff
@@ -89,6 +89,32 @@ quarkus.http.header."Cross-Origin-Opener-Policy".value=same-origin
 quarkus.http.header."Cross-Origin-Resource-Policy".value=same-origin
 quarkus.http.header."Content-Type".value=text/html
 
+# Cache
+# /app for 5min in case hub gets updated
+# /index.html and / for 1min in case hub gets updated
+# /api never because the backend content can change at any time
+# /assets "forever" (1 year) because those files are versioned
+# /favicon.ico and /logo.svg for one day
+quarkus.http.filter.app.header."Cache-Control"=private, max-age=300
+quarkus.http.filter.app.methods=GET,HEAD
+quarkus.http.filter.app.matches=/app/.*
+
+quarkus.http.filter.index.header."Cache-Control"=private, max-age=60
+quarkus.http.filter.index.methods=GET,HEAD
+quarkus.http.filter.index.matches=/index.html|/
```

Comment:
`/app/*` is basically the same as `index.html`, see 

https://github.com/cryptomator/hub/blob/9061af0b2c401cd1ab1c82adacb7179fc4f156dd/backend/src/main/resources/META-INF/web.xml#L19-L26

https://github.com/cryptomator/hub/blob/9061af0b2c401cd1ab1c82adacb7179fc4f156dd/backend/src/main/java/org/cryptomator/hub/filters/VueHistoryModeFilter.java#L11-L21

I guess we can combine both again 😅